src/cmd/usb-util.js

```diff
@@ -124,8 +124,11 @@ async function executeWithUsbDevice({ args, func, dfuMode = false } = {}) {
 	} finally {
 		if (deviceIsProtected) {
 			device = await reopenDevice(device);
-			// FIXME: ignore errors
-			await deviceProtectionHelper.turnOffServiceMode(device);
+			try {
+				await deviceProtectionHelper.turnOffServiceMode(device);
+			} catch (error) {
+				// FIXME: ignore errors
```

Comment:
You can get rid of the FIXME.
Which use case did you see that needs errors here to be ignored?

Reply:
@monkbroc 
`executeWithUsbDevice` is called from `forEachUsbDevice`, and this error will be propagated to `forEachUsbDevice` and if not handled properly, it will show an internal error to the user (such as IN control transfer failed which is likely useless to the user because they cant act on it)

Use case: Enabling Protection on the device immediately after running the safe mode command caused an `IN control transfer failed` error. I added a `waitForDeviceToReboot` after with safe mode command to let it work correctly but I realized that an error (if any) can propagate to `forEachUsbDevice` when using `turnOffServiceMode`

Reply:
Ignoring this error means that an internal error will not be thrown to the user, and device will likely be in Service Mode if any error occurs here that is preventing turning off Service Mode